<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/compiler.ico</normaloff>Images/compiler.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 234, 242);</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>Consolas</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>20</width>
       <height>541</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>471</width>
       <height>541</height>
      </rect>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="token">
      <attribute name="title">
       <string>Token序列</string>
      </attribute>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>461</width>
         <height>471</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QTextBrowser" name="tokenBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QTextBrowser" name="classifyBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>分类显示</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="symbol">
      <attribute name="title">
       <string>符号表</string>
      </attribute>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>511</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>符号总表</string>
        </attribute>
        <widget class="QTextBrowser" name="symboBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>类型表</string>
        </attribute>
        <widget class="QTextBrowser" name="typeBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>数组表</string>
        </attribute>
        <widget class="QTextBrowser" name="ainfBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>结构表</string>
        </attribute>
        <widget class="QTextBrowser" name="rinfBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>形参表</string>
        </attribute>
        <widget class="QTextBrowser" name="fvBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>函数表</string>
        </attribute>
        <widget class="QTextBrowser" name="pfinfBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>常数表</string>
        </attribute>
        <widget class="QTextBrowser" name="consBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>长度表</string>
        </attribute>
        <widget class="QTextBrowser" name="lenBrowser">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>441</width>
           <height>471</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="quarter">
      <attribute name="title">
       <string>四元式</string>
      </attribute>
      <widget class="QTextBrowser" name="quartBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>441</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSlider" name="FontSizeSlider">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>字体大小：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Import"/>
    <addaction name="Export"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="Help"/>
    <addaction name="separator"/>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="StartCompile"/>
    <addaction name="separator"/>
    <addaction name="Clear"/>
    <addaction name="separator"/>
    <addaction name="Restart"/>
   </widget>
   <addaction name="file"/>
   <addaction name="menu"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>宋体</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Import"/>
   <addaction name="separator"/>
   <addaction name="StartCompile"/>
   <addaction name="separator"/>
   <addaction name="Clear"/>
   <addaction name="separator"/>
   <addaction name="Restart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="Import">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/import.png</normaloff>:/image/Images/import.png</iconset>
   </property>
   <property name="text">
    <string>导入源文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Export">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/export.png</normaloff>:/image/Images/export.png</iconset>
   </property>
   <property name="text">
    <string>导出编译结果</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/exit.png</normaloff>:/image/Images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="StartCompile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/start.png</normaloff>:/image/Images/start.png</iconset>
   </property>
   <property name="text">
    <string>开始编译</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="Clear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/clear.png</normaloff>:/image/Images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空输出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="Help">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/help.png</normaloff>:/image/Images/help.png</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="About">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/about.png</normaloff>:/image/Images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="Restart">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/Images/restart.png</normaloff>:/image/Images/restart.png</iconset>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
